pkg/flag/report_flags.go

```diff
@@ -11,12 +11,11 @@ var (
 	FormatYAML  = "yaml"
 	FormatEmpty = ""
 
-	ReportDetectors  = "detectors" // nodoc: internal report type
-	ReportDataFlow   = "dataflow"
-	ReportPrivacy    = "privacy"
-	ReportThirdParty = "thirdparty"
-	ReportSummary    = "summary"
-	ReportStats      = "stats"
+	ReportDetectors = "detectors" // nodoc: internal report type
+	ReportDataFlow  = "dataflow"
```

Comment:
dataflow should be removed from the documentation (could be kept as internal for now) now that we have the privacy report :)

Reply:
@cfabianski would you be happy if we address this in a separate PR, which also adds the privacy report documentation?

Reply:
sure